data/json/items/chemicals_and_resources.json

```
@@ -564,7 +564,7 @@
     "flags": [ "DROP_ACTION_ONLY_IF_LIQUID" ],
     "description": "Sulfuric acid: viscous, foul-smelling and extremely corrosive, particularly to organic matter and evil clowns.  Although used to prepare a number of chemicals, it is mostly known as battery acid, due to its widest commercial application.",
     "volume": "250 ml",
-    "weight": "460 g",
+    "weight": "500 g",
```

Probably safe to keep chemical weights unrounded, not super important though.
```suggestion
    "weight": "460 g",
```